src/include/cpu/power/istep_18.h

```diff
@@ -0,0 +1,634 @@
+#include <cpu/power/scom.h>
+
+void istep_18_11(void);
+void istep_18_12(void);
+
+#define PERV_TOD_ERROR_MASK_REG_RX_TTYPE_0 (38)
+#define PERV_TOD_ERROR_MASK_REG_RX_TTYPE_1 (39)
+#define PERV_TOD_ERROR_MASK_REG_RX_TTYPE_2 (40)
+#define PERV_TOD_ERROR_MASK_REG_RX_TTYPE_3 (41)
+#define PERV_TOD_ERROR_MASK_REG_RX_TTYPE_4 (42)
+#define PERV_TOD_ERROR_MASK_REG_RX_TTYPE_5 (43)
+
+#define PERV_TOD_ERROR_REG_RX_TTYPE_2 (40)
+#define PERV_TOD_ERROR_REG_RX_TTYPE_4 (42)
+#define PERV_TOD_ERROR_REG_RX_TTYPE_5 (43)
+
+#define PERV_TOD_FSM_REG_IS_RUNNING (4)
+#define PERV_TOD_LOAD_TOD_MOD_REG_FSM_TRIGGER (0)
+#define PERV_TOD_START_TOD_REG_FSM_TRIGGER (0)
+#define PERV_TOD_TX_TTYPE_2_REG_TRIGGER (0)
+#define PERV_TOD_TX_TTYPE_4_REG_TRIGGER (0)
+#define PERV_TOD_TX_TTYPE_5_REG_TRIGGER (0)
```

Comment:
If the layout of those registers is the same, do not define a separate macro for each, one is enough.

Reply:
Fixed